process/connections.pb.go

```diff
@@ -785,6 +785,8 @@ type Connection struct {
 	DataStreamsAggregations []byte `protobuf:"bytes,49,opt,name=dataStreamsAggregations,proto3" json:"dataStreamsAggregations,omitempty"`
 	// serialized HTTP2Aggregations object summarizing all http2 transactions recorded for this connection, organized by request path
 	Http2Aggregations []byte `protobuf:"bytes,50,opt,name=http2Aggregations,proto3" json:"http2Aggregations,omitempty"`
+	// serialized map of POSIX error codes to the number of times they have been encountered in the context of this connection (ex [110: 3] would mean that the connection has seen 3 ETIMEDOUT errors)
```

Comment:
Is this PR POSIX specific or is this the general map of error codes across platforms?

Reply:
I propose above that it should be POSIX, and windows maps from the internal windows representation of an error to the POSIX representation of it. This should be pretty easy as for now it's just 3 things. For example `WINTIMEOUT` would be `110`.